tools/misc/calc_timeprop_maxpoldir

```diff
@@ -16,30 +16,37 @@ from scipy import linalg, constants
 import sys
 hplanck = constants.physical_constants['Planck constant in eV s'][0] * 1.0E15
 
-USAGE = """Reads output from TD calculation (after kick) and excitation energy
+USAGE = """%(prog)s [options]
+
+Reads output from TD calculation (after kick) and excitation energy
 (or wavelength) of interest and diagonalizes polarizability tensor, to
 calculate direction of maximum polarizability of that particular excitation.
 
 Needs mux.dat, muy.dat and muz.dat files in working directory."""
 
 def main():
-    parser = argparse.ArgumentParser(description=USAGE)
-    parser.add_argument("tau", metavar="DAMPING",
+    parser = argparse.ArgumentParser(usage=USAGE)
+    parser.add_argument("-d", "--damping", dest="tau",
                         help="damping constant in fs")
-
-    group = parser.add_mutually_exclusive_group(required=True)
-    group.add_argument("-e", "--energy", dest="energy",
+    parser.add_argument("-e", "--energy", dest="energy",
                         help="excitation energy in eV")
-    group.add_argument("-w", "--wavelength", dest="wvlength",
+    parser.add_argument("-w", "--wavelength", dest="wvlength",
                         help="wavelength of excitation in nm")
```

Comment:
Specifiy argument type to float to avoid manual conversion later (and to generate useful error messages, if something else was provided)